config/dev.json

```diff
@@ -15,6 +15,16 @@
       "canGrant": false
     },
     {
+      "id": "38a6b9b3a65a1871",
+      "hashedSecret": "289a885946ee316844d9ffd0d725ee714901548a1e6507f1a40fb3c2ae0c99f1",
```

Comment:
Seems OK to me; looking in the file below, seems like we also took this approach with the `canGrant: true` clients which would also be expected not to hold a secret.